src/Sylius/Behat/Context/Api/Shop/OrderContext.php

```diff
@@ -95,6 +105,36 @@ public function iShouldBeAbleToAccessThisOrderDetails(): void
         );
     }
 
+    /**
+     * @Then /^I should be able to change (payment method on "[^"]+") for (this order)$/
```

Comment:
```suggestion
     * @When /^I change (payment method on "[^"]+") for (this order)$/
```
And then you should assert it with a new payment method, this step only suggests to check the possibility of changing the payment method, but the body changes Method and asserts a new payment method.
WDYT?